x/mint/keeper/keeper.go

```diff
@@ -151,8 +165,30 @@ func (k Keeper) MintCoins(ctx sdk.Context, newCoins sdk.Coins) error {
 	return k.bankKeeper.MintCoins(ctx, types.ModuleName, newCoins)
 }
 
+// GetPoolAllocatableAsset gets the balance of the `MintedDenom` from the `feeCollectorName` module account and returns coins according to the `AllocationRatio`
+func (k Keeper) GetPoolAllocatableAsset(ctx sdk.Context) sdk.Coin {
```

Comment:
Is this correct calculation for GetPoolAllocatableAsset?

Reply:
I don't think this is quite right in that tx fees are also going to the LP's now. (We only want newly minted assets going to them)

I think it should be fine for launch though as this is written, but we should document this.

Reply:
This is still using the feeCollector Module Account.  We need to create a new moduleaccount for `pool_incentives`.

Reply:
It's implemented